public/static/css/common.css

```diff
@@ -1,3 +1,5 @@
+@import url(https://fonts.googleapis.com/css?family=Roboto+Slab:300,400);
```

Comment:
Robot Slab is already imported in base.dt? Where is the appropriate place because I am not a web expert.. At least a double import seem redundant.